ivy_tests/test_ivy/helpers/hypothesis_helpers/number_helpers.py

```diff
@@ -13,6 +13,31 @@
 }
 
 
+@st.composite
+def min_max_bound(draw, min_value=None, max_value=None):
+    min_val = draw(st.just(min_value))
+    max_val = draw(st.just(max_value))
+    return min_val, max_val
+
+
+@st.composite
+def exclude_min_max(draw, exclude_min=True, exclude_max=True):
+    min_ex = draw(st.just(exclude_min))
+    max_ex = draw(st.just(exclude_max))
+    return min_ex, max_ex
+
+
+@st.composite
+def min_max_bound_exclusion(draw,
+                            min_value=None,
+                            max_value=None,
+                            exclude_min=True,
+                            exclude_max=True):
+    min_val, max_val = draw(min_max_bound(min_value=min_value, max_value=max_value))
+    min_ex, max_ex = draw(exclude_min_max(exclude_min=exclude_min, exclude_max=exclude_max))
+    return min_val, max_val, min_ex, max_ex
```

Comment:
I think it's acceptable to stay with the older implemenation of `ints`, as those parameters are limited on what value that can generate, we don't have to add more complexity by using functions that just returns the `st.just` value of the paramter.

Reply:
Hi @CatB1t 
Thankyou for the review and I see your point. But I think this seems a bit needless complexity because almost all existing use of ints strategy had known values specified. Imagine a scenario where you want to generate axes values with the max not exceeding the dimension of arbitrary arrays generated by another shared strategy.  It will be possible to strategize the min max values based on the data being generated alongside ints as shared strategies. Although this can be done in current setting but without the parameter decreasing and I think these sub strategies might have more use than shown here in this PR.
Let me know what you think. 🙂